README.md

```diff
@@ -9,6 +9,8 @@ the add-on works the same in any Kubernetes cluster.
 
 Please see the [documentation site](https://metacontroller.app) for details.
 
+Note: Vendor this package as `k8s.io/metacontroller`.
```

Comment:
IMO this looks a little out of place but I am not sure where this should really go.

Reply:
Isnt there an import comment that tells automated toola how to vebdor/download package with a vanity url?